### DISCUSSION TITLE
Bug: removeChild() not working when using a container from pixi.stage

### DISCUSSION THREAD

**Discussion by steven-twerdochlib** [pixijs/pixijs#10716] [Q&A] [answered]
### Current Behavior

child is not removed.

### Expected Behavior

child to be removed

### Steps to Reproduce

`pixi.stage.removeChild(pixi.stage.children.filter(child =>
          child.name === '123'
        ));`

The above code doesn't remove the child. (I'm certain that the filter is returning one of the children)

### Environment

pixi.js` version: 7.4.2
Browser: Chrome
OS & Version: Windows 10

**Comment by bigtimebuddy** [comment, marked as answer]
Try this using the args spread `...` operator since removeChild doesn't accept an array:

```js
pixi.stage.removeChild(...pixi.stage.children.filter(child => child.name === '123' ));
```
Or alternatively use find for a single element:

```js
pixi.stage.removeChild(pixi.stage.children.find(child => child.name === '123' ));
```


See docs for removeChild:

https://pixijs.download/dev/docs/scene.Container.html#removeChild
